app/controllers/global_issue_templates_controller.rb

```diff
@@ -21,21 +21,20 @@ def index
         @template_map[Tracker.find(tracker.id)] = templates
       end
     end
-    render :layout => !request.xhr?
+    render layout: !request.xhr?
   end
 
   def new
     # create empty instance
     @trackers = Tracker.all
     @projects = Project.all
-    @global_issue_template = GlobalIssueTemplate.new(:author => @user,
-                                        :tracker => @tracker)
+    @global_issue_template = GlobalIssueTemplate.new(author: @user, tracker: @tracker)
```

Comment:
__[rubocop]__
* Line is too long. [86/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)